molecule-runtime/src/commonMain/kotlin/app/cash/molecule/molecule.kt

```diff
@@ -143,7 +146,10 @@ public fun <T> CoroutineScope.launchMolecule(
     val recomposer = Recomposer(coroutineContext)
     val composition = Composition(UnitApplier, recomposer)
     var snapshotHandle: ObserverHandle? = null
-    launch(start = UNDISPATCHED) {
+    launch(
+      context = context,
+      start = UNDISPATCHED
```

Comment:
```suggestion
      start = UNDISPATCHED,
```
I assume ktlint will ask for this too

Reply:
Sorry, I was having some issues getting the gradle build happy with my JDK, addressed [here](https://github.com/cashapp/molecule/pull/312/commits/b5092be63556903e67dc6cc0e8438e56d2aae7a8) (though it looks like this also formatted the whole test file).